src/FeedbackHelpful.ce.vue

```diff
@@ -0,0 +1,291 @@
+              :is-reset="isResetReasons" @reset-completed="handleResetReasonsCompleted" @change="handleReasonsChanged()"
+              :rules="{ required: true }">
+              {{ t("form.infoHardToUnderstand") }}
+            </checkbox-item>
+            <checkbox-item name="reasons" id="infoWrong" :value="Reasons.INFO_WRONG" @change="handleReasonsChanged()">
+              {{ t("form.infoWrong") }}
+            </checkbox-item>
+            <checkbox-item name="reasons" id="helpfulInfoOutdated" :value="Reasons.INFO_OUTDATED"
+              @change="handleReasonsChanged()">
+              {{ t("form.infoOutdated") }}
+            </checkbox-item>
+            <checkbox-item name="reasons" id="infoNotFound" :value="Reasons.INFO_NOT_FOUND"
+              @change="handleReasonsChanged()">
+              {{ t("form.infoNotFound") }}
+            </checkbox-item>
+            <checkbox-item name="reasons" id="helpfulOther" :value="Reasons.OTHER" @change="handleReasonOtherChanged()">
+              {{ t("form.other") }}
+              <div class="form-group" v-if="isReasonOtherSelected">
+                <textarea-input name="comment" id="commentOtherNotHelpful" maxLength.number="750"
+                  :rules="{ required: true, max: 750 }">
+                  {{ t("commentSpecifyDetails") }}
+                </textarea-input>
+                <span class="label label-danger">{{ errors.comment }}</span>
+              </div>
+            </checkbox-item>
+            <span class="label label-danger">{{ errors.reasons }}</span>
+          </fieldset>
+        </fieldset>
+        <!-- Limitation: Can't pass i18n from a custom component to a vue component -->
+        <div>
+          <div class="alert alert-info">
+            <h2 class="h5">{{ t('submissionInformation.note') }}</h2>
+            <ul>
+              <li><small><strong>{{ t('contact.noReply') }}.</strong> {{ t('contact.forEnquiries') }} <a
+                    :href="`${websiteBasePath}${t('contact.opcContactLink')}`">{{ t('contact.contactOpc')
+}}</a>.</small>
+              </li>
+              <li><small><strong>{{ t('submissionInformation.noPersonalInfo') }},</strong> {{
+    t('submissionInformation.personalInfoItems')
+}}.</small></li>
+              <li><small><strong>{{ t('submissionInformation.forMoreInfo') }}</strong>, {{
+    t('submissionInformation.referTo')
+}} <a :href="`${websiteBasePath}${t('submissionInformation.termsOfUseLink')}`">{{
+    t('submissionInformation.termsOfUse')
+}}</a>.</small></li>
+            </ul>
+          </div>
+          <div>
+            <button class="btn btn-primary">{{ t('form.submit') }}</button>
+          </div>
+        </div>
+      </form>
+      <div v-if="submitSuccessful" id="feedbackSuccess">
+        <div>
+          <h2 class="h5">{{ t('confirmation.thankYouForFeedback') }}!</h2>
+          <p>{{ t('contact.noReply') }}. {{ t('contact.forEnquiries') }} <a
```

Comment:
I wonder if https://kazupon.github.io/vue-i18n/guide/interpolation.html#slots-syntax-usage would be cleaner to generate the `p`.